packages/components/organisms/f-loyalty/package.json

```diff
@@ -58,8 +58,6 @@
     "@justeat/f-wdio-utils": "0.3.0",
     "@vue/test-utils": "1.0.3",
     "node-sass-magic-importer": "5.3.2",
-    "@justeat/f-button": "1.9.0",
-    "@justeat/f-card": "1.3.0",
```

Comment:
Are these unnecessary because of how the monorepo works, or are they no longer used?
If the former, could some of the other components below be removed too?

Reply:
I think they weren't used, but this was just a change Maryia put in before I updated the PR so not entirely sure.

@matthewhardern may know more potentially (not sure if Maryia mentioned this change to you?)

Reply:
They are in there as they will be being used soon, I added them as part of the base component as I know I will need them, as this component is in active development and not live.